conda/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set data= load_setup_py_data() %}
+package:
+  name: ts-m2com
+  version: {{ data.get('version') }}
+
+source:
+  path: ../
+
+build:
+  noarch: generic
+  script: {{ PYTHON }} -m pip install --no-deps --ignore-installed .
+
+test:
+  requires:
+    - ts-conda-build
```

Comment:
Please, specify the version of `ts-conda-build`, e.g.:

```
- ts-conda-build =0.3
```

Reply:
Done in the commit: [d6b6a33](https://github.com/lsst-ts/ts_m2com/pull/1/commits/d6b6a33aa2ef307810c241158559d5d3a755cdaa). Thanks!